net/libreswan/files/etc/config/libreswan

```diff
@@ -0,0 +1,37 @@
+config libreswan 'globals'
+	option debug '0' # set debug mode none/all
+	list virtual_private '10.0.0.0/8'
+	list virtual_private '192.168.0.0/16'
+	list virtual_private '172.16.0.0/12'
+	list virtual_private '25.0.0.0/8'
+	list virtual_private '100.64.0.0/10'
+	list virtual_private '!100.64.0.0/24'  # the address ranges that may live behind a NAT router through which a client connects
+	# option listen '192.168.2.100' # listening address, if set listen_interface would not be used
+	# option listen_interface 'wan' # listening interface
+	# option uniqueids 'yes' # yes/no
+
+config crypto_proposal 'p1'
```

Comment:
without luci, it's not intuitive hot you attach the proposals in the config file

Reply:
shouldn't multiple encryption methods also be available on one proposal ? or multiple proposals can be attached to one interface?

Reply:
Added support to handle multiple encryptions in UCI and also from LuCI.